components/switch/README.md

```diff
@@ -61,7 +61,7 @@ To make your usage of `d2l-switch` accessible, use the following property:
 
 ## Visibility Switch [d2l-switch-visibility]
 
-The `d2l-switch-visibility` element is a variant of the generic switch configured with special icons and default text for toggling "visibility".
+The `d2l-switch-visibility` element is a variant of the generic switch configured with special icons and default text for toggling "visibility".  
```

Comment:
Seems like you missed some extra whitespace here

Reply:
Oh, yeah, I added that in there so that the sentence after it would properly render on a new line on the page 
(I think it's a markdown quirk where you need at least two spaces to have the next sentence show up on a new line)

Reply:
@dcostiuc you should use a blank newline between lines of text if you want them to appear on separate lines.